common/src/main/java/com/aliyun/emr/rss/common/network/server/BaseHandler.java

```diff
@@ -24,14 +24,12 @@
 
 import com.aliyun.emr.rss.common.network.client.RpcResponseCallback;
 import com.aliyun.emr.rss.common.network.client.TransportClient;
-import com.aliyun.emr.rss.common.network.protocol.PushData;
-import com.aliyun.emr.rss.common.network.protocol.PushMergedData;
 import com.aliyun.emr.rss.common.network.protocol.RequestMessage;
 
 /**
  * Handler for sendRPC() messages sent by {@link TransportClient}s.
  */
-public abstract class RpcHandler {
+public class BaseHandler {
```

Comment:
If it's a class, call `BaseHandler` is a little strange..

Reply:
How about BaseMessageHandler?